src/ruleconfig.jl

```diff
@@ -0,0 +1,21 @@
+using ChainRulesCore: RuleConfig, rrule_via_ad
```

Comment:
It's not a big deal, but I prefer imports to be in the main module file.

Reply:
done